error using cloudwatch output writer

Hi,

I'm keen to use jmxtrans to output metrics to cloud watch.   Trying the example here: 
https://github.com/jmxtrans/jmxtrans/wiki/CloudWatchWriter

On Centos7, with RPM: http://central.maven.org/maven2/org/jmxtrans/jmxtrans/251/jmxtrans-251.rpm

with only changes to the host, alias and port, I get a stack trace in the logs.  In short, is says:

Caused by: com.fasterxml.jackson.databind.JsonMappingException: Conflicting setter definitions for property "region": com.amazonaws.AmazonWebServiceClient#setRegion(1 params) vs com.amazonaws.AmazonWebServiceClient#setRegion(1 params) (through reference chain: com.googlecode.jmxtrans.model.JmxProcess["servers"]->java.util.ArrayList[0]->
java.util.ArrayList[0]->java.util.ArrayList[0])

Short version here: http://pastebin.com/Apk6ZSWs

I can provide a full trace if needed.

The config file I have tried is just a simple test:
```json
{
   "servers":[
      {
         "alias":"testhost",
         "host":"localhost",
         "port":"1105",
         "queries":[
            {
               "obj":"java.lang:type=OperatingSystem",
               "attr":[
                  "SystemLoadAverage"
               ],
               "resultAlias":"jvm.cpu",
               "outputWriters":[
                  {
                     "@class":"com.googlecode.jmxtrans.model.output.CloudWatchWriter",
                     "settings":{
                        "namespace":"jmx"
                     }
                  }
               ]
            }
         ]
      }
   ]
}
```

--- a/jmxtrans-output/jmxtrans-output-cloudwatch/src/main/java/com/googlecode/jmxtrans/model/output/CloudWatchWriter.java
+++ b/jmxtrans-output/jmxtrans-output-cloudwatch/src/main/java/com/googlecode/jmxtrans/model/output/CloudWatchWriter.java
@@ -29,6 +29,7 @@ import com.amazonaws.services.cloudwatch.AmazonCloudWatchClient;
 import com.amazonaws.services.cloudwatch.model.MetricDatum;
 import com.amazonaws.services.cloudwatch.model.PutMetricDataRequest;
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.annotations.VisibleForTesting;
 import com.google.common.base.MoreObjects;
@@ -74,6 +75,7 @@ public class CloudWatchWriter extends BaseOutputWriter {
 		if (isNullOrEmpty(this.namespace)) throw new IllegalArgumentException("namespace cannot be null or empty");
 	}
 
+	@JsonIgnore
 	@VisibleForTesting
 	void setCloudWatchClient(AmazonCloudWatchClient cloudWatchClient) {
 		this.cloudWatchClient = cloudWatchClient;

--- a/jmxtrans-output/jmxtrans-output-cloudwatch/src/test/java/com/googlecode/jmxtrans/model/output/CloudWatchWriterTests.java
+++ b/jmxtrans-output/jmxtrans-output-cloudwatch/src/test/java/com/googlecode/jmxtrans/model/output/CloudWatchWriterTests.java
@@ -22,11 +22,17 @@
  */
 package com.googlecode.jmxtrans.model.output;
 
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+
 import com.amazonaws.services.cloudwatch.AmazonCloudWatchClient;
 import com.amazonaws.services.cloudwatch.model.MetricDatum;
 import com.amazonaws.services.cloudwatch.model.PutMetricDataRequest;
 import com.google.common.collect.ImmutableList;
 import com.google.common.collect.ImmutableMap;
+import com.googlecode.jmxtrans.util.JsonUtils;
+import com.googlecode.jmxtrans.model.JmxProcess;
 import com.googlecode.jmxtrans.model.Query;
 import com.googlecode.jmxtrans.model.Result;
 import com.googlecode.jmxtrans.model.Server;
@@ -87,6 +93,13 @@ public class CloudWatchWriterTests {
 		MetricDatum metricDatum = request.getMetricData().get(0);
 		assertThat(metricDatum.getMetricName()).isEqualTo("attributeName_key");
 		assertThat(metricDatum.getValue()).isEqualTo(1);
+	}
+
+	@Test
+	public void loadingFromFile() throws URISyntaxException, IOException {
+		File input = new File(CloudWatchWriterTests.class.getResource("/cloud-watch.json").toURI());
+		JmxProcess process = JsonUtils.getJmxProcess(input);
+		assertThat(process.getName()).isEqualTo("cloud-watch.json");
 	}
 
 }

--- a/jmxtrans-output/jmxtrans-output-cloudwatch/src/test/resources/cloud-watch.json
+++ b/jmxtrans-output/jmxtrans-output-cloudwatch/src/test/resources/cloud-watch.json
@@ -1,0 +1,26 @@
+{
+   "servers":[
+      {
+         "alias":"testhost",
+         "host":"localhost",
+         "port":"1105",
+         "queries":[
+            {
+               "obj":"java.lang:type=OperatingSystem",
+               "attr":[
+                  "SystemLoadAverage"
+               ],
+               "resultAlias":"jvm.cpu",
+               "outputWriters":[
+                  {
+                     "@class":"com.googlecode.jmxtrans.model.output.CloudWatchWriter",
+                     "settings":{
+                        "namespace":"jmx"
+                     }
+                  }
+               ]
+            }
+         ]
+      }
+   ]
+}

